src/App/App.tsx

```diff
@@ -72,6 +74,7 @@ function newMetabaseTab(t: TFunc): TabData {
     type: 'metabase',
     stale: false,
     refreshNonce: 0,
+    startUrlPath: 'browse/3-anonymized-access',
```

Comment:
this is a bit of a placeholder url. It's a bit better than the `/` one - it points directly to the page showing all tables, so it's easy for the user to find and click on the table just imported.

Subject to change.

Reply:
Is `3` the id of the `anonymized-access` data source? If so, can it be made into constant instead of using the opaque literal value?

Reply:
I'm changing this right now in a follow-up, this requires some more figuring out (Ideally, it should be discovered from the API somehow, I suppose).

I can hold off merging this, if you think it might lead to unexpected problems, like when anonymized access ends up not being `3` or sth, but in general this is temporary

Reply:
Up to you... It doesn't bother me leaving as it is for a short while.

Reply:
We will need proper data source and table IDs for examples. Ok to leave the hack for now.

Reply:
Hm, I'm digging in to this very thing right now. There's a way to discover all this data in a series of API requests and filters. It might generate a lot of back-and-forth between dashboards and mb, but could spare us the hassle of storing more state

Reply:
I'm thinking of using metabase as the source of truth instead of posgres. Meaning we query `/api/database?include=tables` to get the list of tables and their IDs.

Reply:
yup, this is what I had in mind above